README.md

```diff
@@ -2,28 +2,50 @@
 
 [![Build Status](https://travis-ci.org/deis/workflow-cli.svg?branch=master)](https://travis-ci.org/deis/workflow) [![Go Report Card](http://goreportcard.com/badge/deis/workflow-cli)](http://goreportcard.com/report/deis/workflow-cli)
 
-
 `deis` is a command line utility used to interact with the [Deis](http://deis.io) open source PaaS.
 
-Please add any [issues](https://github.com/deis/workflow/issues) you find with this software to
-the [Deis Workflow v2 Project](https://github.com/deis/workflow).
+Please add any [issues](https://github.com/deis/workflow-cli/issues) you find with this software to
+the [Deis CLI Project](https://github.com/deis/workflow-cli).
+
 
 ## Installation
 
-Currently the only way to use the go version of the `deis` client is to build it yourself.
-To build the `deis` client, you need to have [go](https://golang.org/), [glide](https://github.com/Masterminds/glide),
- and [make](https://www.gnu.org/software/make/) installed. Then run `make build`.
+### From Bintray
+
+Install the `deis` client from [bintray](https://bintray.com/) by running
+
+	$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash
+
+The installer will fetch the latest version of the client into your current directory.
+
+	$ ./deis --version
```

Comment:
@bacongobbler maybe add some language here about moving the binary into your `PATH`?